docs/sources/project/test-and-docs.md

```diff
@@ -249,7 +249,7 @@ can browse the docs.
 
 1. In a terminal, change to the root of your `docker-fork` repository.
 
-        $ cd ~/repos/dry-run-test
+        $ cd ~/repos/docker-fork
```

Comment:
In this case, the dry-run-test is correct it is the step instructions that are wrong: 

In a terminal, change to the root of your `dry-run-test` repository.